flow/flat_buffers.h

```diff
@@ -233,13 +231,13 @@ template <class T>
 struct sfinae_true : std::true_type {};
 
 template <class T>
-auto test_deserialization_done(int) -> sfinae_true<decltype(T::deserialization_done)>;
+auto test_serialization_done(int) -> sfinae_true<decltype(T::serialization_done)>;
```

Comment:
I like this one 😉 

Though maybe add a short comment how this SFINAE function works.

Reply:
The funny thing is that I plan to remove this when implementing IReplicationPolicy with the new dynamic_size_traits idea, since this is mostly a hack to work around ownership issues.

Done.